### DISCUSSION TITLE
Preserve files modification time

### DISCUSSION THREAD

**Discussion by drovnotya** [complexlogic/rsgain#78] [Ideas]
It's too late for my own use because I have run rsgain on my files already, but if this was an option I would have used it.

For instance metaflac has a --preserve-modtime, and also foobar2000 and Mp3tag (Windows software) has the ability to do the same when altering a files tags.

**Comment by complexlogic** [comment]
I'll look into this.

**Comment by complexlogic** [comment]
Implemented via https://github.com/complexlogic/rsgain/commit/09f84d82dbfd3f9ec2775cb8b690b8bcc21caf49
